app/src/routes.js

```diff
@@ -11,7 +11,7 @@ export default (store) => {
   return [
     {
       path: `/`,
-      redirect: `/cosmos-hub/portfolio`,
+      redirect: `/notifications`,
```

Comment:
this is the simplest way to make notifications the home page. kind of radical - but also makes a lot of sense. 

if the user is not signed in they will be redirected to the paywall which is handled by the router guard. 

this will also really help with signups and exposure for notifications.

Reply:
Mmm, I think this is too radical until we don't have really figured the final flow out. We can do this once we merge the "traditional" sign in with the new email-based one. But doing this right now is a bad idea (too confusing)

Reply:
> merge the "traditional" sign in with the new email-based one.

what do you mean by this?

currently, we default to cosmos hub profile page. this is not really "not confusing" it's just how it's been for so long.

Reply:
We agreed, that if the user is not signed in, we forward him to the validators page not let him be stuck on notifications.

Reply:
no no no we agreed that if the email modal closes and they are on a page that requires an address that we would send them to the validators page. we didn't discuss changing the landing page for non-email users.

anyways. we'll revert for now.